django_project/base_user/models/__init__.py

```diff
@@ -0,0 +1,4 @@
+__author__ = 'Irwan Fathurrahman <irwan@kartoza.com>'
+__date__ = '26/02/18'
+
+from .user import User
```

Comment:
You need to write `from base_user.models.user import User` to fix the error test.